tests/bc/test_bc.cu

```diff
@@ -40,53 +40,52 @@ struct main_struct
         typename VertexT, // Use int as the vertex identifier
         typename SizeT,   // Use int as the graph size type
         typename ValueT>  // Use int as the value type
-    cudaError_t operator()(util::Parameters &parameters, VertexT v, SizeT s, ValueT val)
+    cudaError_t operator()(util::Parameters &parameters,
+        VertexT v, SizeT s, ValueT val)
     {
-        typedef typename app::TestGraph<VertexT, SizeT, ValueT, graph::HAS_EDGE_VALUES | graph::HAS_CSR> GraphT; // PR different
-        
+        typedef typename app::TestGraph<VertexT, SizeT, ValueT, graph::HAS_CSR> 
+            GraphT;
+
         cudaError_t retval = cudaSuccess;
         util::CpuTimer cpu_timer;
         GraphT graph;
 
-        parameters.Set("undirected", true);
-
         cpu_timer.Start();
         GUARD_CU(graphio::LoadGraph(parameters, graph));
         cpu_timer.Stop();
         parameters.Set("load-time", cpu_timer.ElapsedMillis());
 
-        // Enable is set sources        
-        // GUARD_CU(app::Set_Srcs(parameters, graph));
+        // Enable is set sources
+        GUARD_CU(app::Set_Srcs(parameters, graph));
         int num_srcs = 0;
 
-        // TODO: reference result on CPU, e.e.:
-        ValueT **reference_bc_values    = NULL;
-        ValueT **reference_sigmas       = NULL;
-        VertexT **reference_source_path = NULL; 
-        
+        ValueT  **reference_bc_values    = NULL;
+        ValueT  **reference_sigmas       = NULL;
+        VertexT **reference_source_path = NULL;
```

Comment:
Are we using a linter?  Or is there a style guide dictating this kind of spacing?

Reply:
We don't have an official style guide, as long as the style is not too different from the existing code. Most of the time I align the code so it's easier to see the common and the different parts of neighboring lines.

Reply:
OK makes sense -- if there are going to be lots of contributors, it could be good to think about using a linter, to reduce the number of whitespace diffs.  Might be a huge pain to implement retrospectively, but could be something worth thinking about for new apps like this one.